controllers/comments.js

```diff
@@ -192,25 +198,35 @@ export default {
         });
       }
 
+      const findComment = await Comments.findOne({ where: { id: commentId } });
+      if (!findComment) {
+        return res.status(404).send({
+          status: res.statusCode,
+          error: 'The comment is not found'
+        });
+      }
+      const isAuthor = !!(id === CommentedArticle.author && role === 'admin');
+      if (!isAuthor && id !== findComment.author) {
```

Comment:
I think here you also allowed the article author to be able to change comments? 🤔🤔I think that we were against this idea?

Reply:
Thank you @avpaul for pointing this out. Now it is good.